app/presenters/tree_builder_alert_profile_obj.rb

```diff
@@ -0,0 +1,53 @@
+class TreeBuilderAlertProfileObj < TreeBuilder
+  def initialize(name, type, sandbox, build = true, assign_to: nil, cat: nil, objects: nil)
+    @assign_to = assign_to
+    @cat = cat
+    @objects = objects
```

Comment:
The `objects` and `@objects` variables are a little confusing, can you rename them to something more descriptive?